doc/whats_new/v1.2.rst

```diff
@@ -502,6 +502,13 @@ Changelog
   :func:`utils.discovery.all_functions`) in scikit-learn.
   :pr:`21469` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |API| Added an `"auto"` option to the `normalized_stress` argument in
+  :class:`manifold.MDS` and :func:`manifold.smacof`. Note that
+  `normalized_stress` is only valid for non-metric MDS, therefore the `"auto"`
+  option enables `normalized_stress` when `metric=False` and disables it when
+  `metric=True`.
+  :pr:`23834` by :user:`Meekail Zain <micky774>`
```

Comment:
I guess the what's new should also mention that "auto" will become the default in 1.4